gui/builtinPreferenceViews/pyfaNetworkPreferences.py

```diff
@@ -111,6 +112,21 @@ def populatePanel( self, panel ):
 
         mainSizer.Add( fgAddrSizer, 0, wx.EXPAND, 5)
 
+        # proxy auth information: login and pass
+        self.stPSetLogin = wx.StaticText(panel, wx.ID_ANY, u"Proxy login:", wx.DefaultPosition, wx.DefaultSize, 0)
+        self.stPSetLogin.Wrap(-1)
+        self.editProxySettingsLogin = wx.TextCtrl(panel, wx.ID_ANY, self.nAuth[0], wx.DefaultPosition, wx.DefaultSize, 0)
+        self.stPSetPassword = wx.StaticText(panel, wx.ID_ANY, u"pass:", wx.DefaultPosition, wx.DefaultSize, 0)
```

Comment:
This is nitpickey, but can we change the labels to simply say "Username" and "Password"? :D